index.js

```diff
@@ -349,6 +349,23 @@ server.route([
     }
   },
   {
+    method: 'POST',
+    path: '/properties/{propertyId}/calendarEvents',
+    config: {
+      handler: require('./handlers/propertyCalendarEvents.js'),
+      auth: 'session',
+      validate: {
+        payload: {
+          eventDesc: Joi.string().required(),
+          date: Joi.date().format('DD/MM/YYYY'),
+          time: Joi.date().format('h:mm a'),
+          notes: Joi.optional(),
```

Comment:
You have imposed a constraint in the DB schema on the length of both `notes` and `eventDesc`. You should reflect those constraints here. Simple case, if I write a big note with >1024 characters, everything greater than 1024 characters will simply be deleted. This is a general habit to be in when programming a system. If you introduce an artificial limit (eg 1024 characters) every part of the system should reflect this limit. Otherwise you just litter the system with silent failures and bugs, which over time gets really annoying. And these things are typically easy to prevent when you introduce these constraints. "Oh I just made a hard limit on how many characters you can add, better make sure the input validation lets someone know this is the case so it doesn't silently fail on them".